Tribler/Policies/BoostingManager.py

```diff
@@ -0,0 +1,493 @@
+# -*- coding: utf-8 -*-
+# Written by Egbert Bouman, Mihai Capotă, Elric Milon, and Ardhi Putra Pratama H
+"""Manage boosting of swarms"""
+import logging
+import os
+import shutil
+from binascii import hexlify, unhexlify
+
+import libtorrent as lt
+from twisted.internet.task import LoopingCall
+
+from Tribler.Core.DownloadConfig import DownloadStartupConfig, DefaultDownloadStartupConfig
+from Tribler.Core.Libtorrent.LibtorrentDownloadImpl import LibtorrentDownloadImpl
+from Tribler.Core.Utilities import utilities
+from Tribler.Core.Utilities.configparser import CallbackConfigParser
+from Tribler.Core.exceptions import OperationNotPossibleAtRuntimeException
+from Tribler.Core.simpledefs import DLSTATUS_SEEDING, NTFY_TORRENTS, NTFY_UPDATE, NTFY_CHANNELCAST
+from Tribler.Policies.BoostingPolicy import SeederRatioPolicy
+from Tribler.Policies.BoostingSource import ChannelSource
+from Tribler.Policies.BoostingSource import DirectorySource
+from Tribler.Policies.BoostingSource import RSSFeedSource
+from Tribler.Policies.credit_mining_util import source_to_string, string_to_source, compare_torrents
+from Tribler.Policies.defs import SAVED_ATTR, CREDIT_MINING_FOLDER_DOWNLOAD, CONFIG_KEY_ARCHIVELIST, \
+    CONFIG_KEY_SOURCELIST, CONFIG_KEY_ENABLEDLIST, CONFIG_KEY_DISABLEDLIST
+from Tribler.dispersy.taskmanager import TaskManager
+
+
+class BoostingSettings(object):
+    """
+    Class contains settings on boosting manager
```

Comment:
"This class contains settings used by the boosting manager"

(please make sure to use correct English in your docstrings since these might eventually end up in a generated documentation file)